src/main/java/com/minecolonies/rankup/util/Constants.java

```diff
@@ -0,0 +1,34 @@
+package com.minecolonies.rankup.util;
+
+/**
+ * Constants for our plugin.
+ */
+public final class Constants
+{
+
+    private Constants()
+    {
+        //Empty private constructor.
+    }
+
+    public final class PlayerInfo
+    {
+        public static final String PLAYER_NAME = "{player}";
+        public static final String PLAYER_GROUP = "{group}";
+        public static final String PLAYER_RANK = "{rank}";
+        public static final String PLAYER_PREFIX = "{prefix}";
+        public static final String PLAYER_JOIN = "{joindate}";
+        public static final String PLAYER_LAST = "{lastjoin}";
+        public static final String PLAYER_TRACK = "{track}";
+    }
+
+    public final class ModuleInfo
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this a "static" inner class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2694)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1118)